packages/sync-actions/test/category-sync.spec.js

```diff
@@ -232,6 +232,16 @@ describe('Actions', () => {
       ]
       expect(actual).toEqual(expected)
     })
+
+    test('should return empty array', () => {
```

Comment:
Nice test 👍  But may be we can find a more descriptive name? May be `no exception thrown on missing assets`?

Reply:
Consider it done